ophyd/sim.py

```diff
@@ -649,19 +638,6 @@ def __init__(self, *args, reg=DO_NOT_USE, save_path=None, save_func=np.save,
         else:
             self.reg = None
 
-    @property
-    def hints(self):
-        if self._hints is None:
-            # Since data is external, hint that it should not be printed or
-            # plotted. Relax this when LiveTable etc. get smarter about
```

Comment:
Per this comment, I _think_ LiveTable and friends are now smart enough to deal with this.